base.docker

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-slim-buster
+FROM python:3.8-slim-buster
```

Comment:
Exciting! We must remember to bump the Python version on all the Travis checks as well

Reply:
Removed this in favour of doing in a separate commit